### ISSUE TITLE
Cant reproduce demo app

### ISSUE THREAD

**Issue by vach** [edvin/tornadofx#137]
In your [video demonstration](https://www.youtube.com/watch?v=rjc8_HGHy3c) when defining mixin there is something that does not work in latest version.

http://imgur.com/zKTpa0D

**Comment by edvin** [comment]
There has been significant improvements to the CSS support, but to make it happen we had to break backwards compatibility in a couple of places. You can read about these breaking changes in the release notes for version 1.5.0:

https://github.com/edvin/tornadofx/releases/tag/v1.5.0

Specifically this has to do with supporting "multi values", ie. CSS properties that can have multiple values. If you just want to add a single value to a multi value property you do:

```kotlin
backgroundInsets += box(0.px)
```

So basically, add a `+` and you're good to go :) I'm going to make a new video demonstrating the new syntax soon.

**Comment by vach** [comment]
Awesome, people shall do video tutorials more, that way you can just add a new video, delete old one and all new users will be on latest version... No idea why many companies don't do this... Jetbrains seems to be on the right path though...
